private/bufpkg/bufcheck/buflint/testdata/validate_rules_types_match/a.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package a;
+
+import "buf/validate/validate.proto";
+
+message Foo {
```

Comment:
What is the linting we want to do for non-primitives (`enum, repeated, map, any, duration, timestamp`)? This rule doesn't take those into account.

Reply:
these have not been implemented yet, this will be completed before the PR is ready for review